### ISSUE TITLE
Structural hierarchy

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/epub-specs#2913]
### Section

Structural hierarchy

### Describe the problem

I have some doubts about this section: we are actually recommending the use of semantic tags (which is one of the basic rules of accessibility), so perhaps it was included because some authoring tools do not handle them correctly?

### Describe the fix or new feature you propose

I suggest removing it.

**Comment by clapierre** [comment]
I agree this should be removed. 
There is also a typo "... we pull can this structure..."
And we should not include class="paragraph" on a <p> tag.

I also recommend removing this section "3.2.2.4 Structural hierarchy" from the Fixed Layout Techniques document.

**Comment by clapierre** [comment]
We may need additional guidance for Fixed Layout Page titles since in a fixed layout picture book each image could be on a separate page. What should we recommend how to uniquely label these pages?

Dale Rogers: why would any xhtml doc not have a title.
… its a required element.

Gregorio Pellegrino: fixed layout each page is and xhtml file and each file has to have a title.
… page 2 is page after title page usually blank or illistration.
… what do we tell the publishers what is the title of page 2 thats blank?
… title of story, what do you put for the 2nd page title.

Dale Rogers: right need guidelines.

Avneesh Singh: Refer to the title section of EPUB Accessibility Techniques 1.2 and determine what specific guidance we want to provide for fixed layout techniques.

**Comment by avneeshsingh** [comment]
copying discussions held in Feb 19 meeting for record purpose:
Gregorio P.:… We should refer to EPUB Accessibility Techniques 1.1 because it is a stable document.
… Challenges include discussion of reading order.
… We need to determine which document will address reading order techniques.

Avneesh Singh: Do we have relevant WCAG techniques that we can link here?

Gregorio P.: Yes, EPUB Accessibility Techniques point to some WCAG guidance.

AvneeshSingh: So, we should refer to EPUB Accessibility Techniques 1.2 and it already includes links to WCAG guidance.
